lib/tests/unit/configuration_test.cc

```diff
@@ -435,6 +437,30 @@ TEST_CASE("Configuration::parseOptions duplicate broker-ws-uris", "[configuratio
     }
 }
 
+TEST_CASE("Configuration::parseOptions invalid config-file name", "[configuration]") {
+    const char* altArgv[] = {
+    "test-command",
+    "--config-file", INVALID_CONFIG_NAME.data(),
```

Comment:
We should use `c_str()` instead of `data()` as it guarantees nul terminated string which is what we want here IMO.

Reply:
Ok.